src/main/java/org/futurecollars/lesson2task5evenOddNumbers/EvenOddNumbers.java

```diff
@@ -0,0 +1,11 @@
+package org.futurecollars.lesson2task5evenOddNumbers;
+
+public class EvenOddNumbers {
+
+    public static boolean isEven (int a) {
+        return a % 2 == 0;
+    }
+    public static boolean isOdd (int a){
+        return !(a % 2 == 0);
```

Comment:
Gdybyś zamiast " !(a % 2 == 0)" dał "a % 2 != 0" to było by krócej i miałbyś jedną operację mniej. Ale to tak na przyszłość.